docs/usage/casting.md

```diff
@@ -1,57 +1,121 @@
-# Casting the Card
+# Casting
 
-This card can be (Chrome) casted to a device (such as a [Nest
-Hub](https://store.google.com/us/product/nest_hub_2nd_gen)) through the use of
-[Home Assistant Cast](https://cast.home-assistant.io/).
+This card allows casting a camera stream to a Google Cast device. You can enable the cast button with:
 
-## Instructions
+```yaml
+type: custom:frigate-card
+cameras:
+  - camera_entity: camera.office
+    live_provider: go2rtc
+menu:
+  media_player:
```

Comment:
@dermotduffy, this made me realize that the button would be better named `cast` rather than `media_player`.

Reply:
Think it ended up media_player since that is what HA calls the entities (e.g. `media_player.tv`) & services (e.g. `media_player.media_play`). I assume this is because technically, the actual underlying `media_player` might be using something entirely unrelated to "Google Cast" to play the media.